src/app/shared/components/navbar-mobile/navbar-mobile-dialog/navbar-mobile-dialog.component.html

```diff
@@ -1,13 +1,21 @@
 <mat-card class="navbar-mobile__dialog">
   <mat-card-header class="navbar-mobile__dialog__header">
-    <a (click)="this.startLogin()" *ngIf="!this.isAuthenticated; else userProfile" class="navbar__item" mat-button>Login</a>
+    <a (click)="this.startLogin()" *ngIf="!this.isAuthenticated; else userProfile" class="navbar__item navbar__user" mat-button>
+      <mat-icon svgIcon="ktzh_user"></mat-icon>
+      <span>Login</span>
```

Comment:
Also: I've quickly tested this by navigating only using the keyboard. I'm not able to reach the `login` or the logged in user using tab or the arrow keys. Could also be _skill issues_ on my part 😅

Reply:
True, I changed it to be a button (same as on desktop nav), so it is no reachable via tab